community/portworx/templates/portworx-ds.yaml

```diff
@@ -26,7 +26,7 @@
 {{- $etcdEndPoints := .Values.kvdb }}
 {{- $misc := .Values.misc | default "" | split " " }}
 
-apiVersion: apps/v1
+apiVersion: extensions/v1beta1
```

Comment:
The last merge failed to install Portworx because of these apiversion's, that is why we are reverting them back also, we cannot use apps/v1 or extensions/v1 for a DaemonSet, it gives the following error
```
error: error validating "spec123.yaml": error validating data: ValidationError(DaemonSet.spec): missing required field "selector" in io.k8s.api.apps.v1.DaemonSetSpec; if you choose to ignore these errors, turn validation off with --validate=false

Error: unable to recognize "": no matches for kind "DaemonSet" in version "extensions/v1"
```
Following suggestion in this document: [https://matthewpalmer.net/kubernetes-app-developer/articles/kubernetes-apiversion-definition-guide.html](https://matthewpalmer.net/kubernetes-app-developer/articles/kubernetes-apiversion-definition-guide.html)